tasks/FileUploadTask.md

```diff
@@ -16,9 +16,26 @@ This task aims to provide a fluent and easy to use mechanism for the consumer to
 - The task should signal upload completion via by returning the status or returning a completed **Task**/**Promise**/**Future** for async APIs.
 - Using the RequestContext, the feature flag for the **FileUploadTask** can be set for telemetry purposes.
 - The task should not retry to upload failed slices as any retry should already be done by the retry-handler which the task should be using.
-- The task should provide cancellation capabilities through native task cancellation sources when using async APIs.
-- For Outlook, OneDrive and PrintDocument APIs, send a delete request to the upload session URL to cancel an upload task. 
+- Upload task cancellation - 
+  - The task should provide cancellation capabilities through native task cancellation sources when using async APIs.
+  - For Outlook, OneDrive and PrintDocument APIs, send a delete request to the upload session URL to cancel an upload task. 
 - The task should provide api to access the upload session object containing its URL, expiry date and the cancellation status.
+  - Implement `getUploadSession()` which returns a `LargeFileTaskUploadSession` object.
+  - `LargeFileTaskUploadSession` -
+    ```json
+    "properties": {
```

Comment:
Can you move that in a dedicated section like I did for the result prototype please? it'll make the list of requirements easier to parse. https://github.com/microsoftgraph/msgraph-sdk-design/blob/7552dacea815db48b11c179bd764f6d0e096ea92/tasks/FileUploadTask.md#large-file-upload-result-prototype

Reply:
Done.